app/models/jobseeker_profile.rb

```diff
@@ -120,7 +120,8 @@ def qts_status
   end
 
   def activable?
-    !!personal_details&.complete? && !!job_preferences&.complete?
+    personal_details&.complete? && job_preferences&.complete? &&
```

Comment:
although nil is falsey should it return false in that case as a predicate method